schedule/yast/sle/guided_xfs/guided_xfs.yaml

```diff
@@ -2,9 +2,18 @@
 name:           guided_xfs
 description:    >
   Guided Partitioning installation with xfs filesystem.
+# Note: Test data for different backends need to be
```

Comment:
we don't need this comment anywhere, this will probably be obvious and documented when we try to have a single test suite in the next iterations.
It is not easy to say that this would work fine with other arch/backends, unless you want to provide extra verification, the easiest way is to create a new file for now rather than editing this one. The plan is to analyze test suite per test suite to have one yaml file and then we will be providing VR for each arch/backend to see that is consistent, even with openSUSE.